src/iohumdrum.cpp

```diff
@@ -20228,6 +20228,11 @@ Beam *HumdrumInput::insertBeam(
     std::vector<std::string> &elements, std::vector<void *> &pointers, const humaux::HumdrumBeamAndTuplet &tg)
 {
     Beam *beam = new Beam();
+    if (tg.token->find("yy") != std::string::npos) {
+        // Ignore beam when token is suppressed with yy signifier
+        beam->SetType("invisible");
+        beam->SetColor("transparent");
```

Comment:
Any ideas how to improve this?

Reply:
I think this is the best way.   This is equivalent to `visibility: none` in CSS: space will be allocated to the beams but they will not be shown.   This should be sufficient as long as you don't mind the extra vertical space between staves that this may create.